app/views/teachers/classes/TeacherStudentView.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <loading-progress :loading-status="[ backboneLoadProgress ]" :always-render="true">
```

Comment:
I think I am confused about what `backboneLoadProgress` can be and the way it is handled.

Reply:
Actually this looks fine. See my comments in the the `LoadingProgress` component.